cmd/doc/build.rs

```diff
@@ -48,6 +48,7 @@ fn main() -> Result<()> {
     write!(
         output,
         r##"
+#[allow(clippy::needless_raw_string_hashes)]
```

Comment:
I don't think generated code needs to worry about how many hashes surround its strings.